src/css/components/modal.less

```diff
@@ -5,77 +5,103 @@
 .modal {
   display: block;
   border-radius: @modal-border-radius;
-}
+  color: @modal-dialog-text-color;
 
-.modal-header {
-  background-color: @sidebar-bg-color;
-  border-bottom: none;
-  border-radius: @modal-border-radius @modal-border-radius 0 0;
-}
+  dd {
+    color: inherit;
+  }
 
-.modal-footer {
-  border-radius: 0 0 @modal-border-radius @modal-border-radius;
-  border-top: none;
-  display: flex;
-  margin-top: 0;
-  justify-content: center;
-  padding: @modal-footer-padding;
-}
+  .modal-header {
+    border-bottom: none;
+    border-radius: @modal-border-radius @modal-border-radius 0 0;
+    background-color: @modal-dialog-info-color;
+    color: @modal-header-text-color;
+    max-height: @modal-header-height;
 
-.modal-content {
-  background-color: @primary-bg-color;
-}
+    padding: @modal-header-padding;
 
-.modal-header-blend {
-  background: transparent;
-}
+    .modal-title {
+      line-height: 1;
+    }
+  }
+
+  .modal-content {
+    background-color: @modal-dialog-bg-color;
+    border: none;
+    box-shadow: none;
+    min-width: @modal-dialog-width;
+  }
 
-.modal-body {
-  word-wrap: break-word;
+  .modal-body {
+    padding: 0 @base-spacing-unit*2.5;
 
-  .row.full-bleed {
-    margin-left: -@modal-body-padding;
-    margin-right: -@modal-body-padding;
-    .panel-heading {
-      padding-left: @modal-body-padding;
-      padding-right: @modal-body-padding;
-    }
-    .panel-body {
-      padding-left: @modal-body-padding;
-      padding-right: @modal-body-padding;
-    }
-    &.reduced-padding {
-      .panel-body {
-        padding-left: 0px;
-        padding-right: 0px;
+    max-height: calc(~"80vh" - (@modal-header-height+@modal-footer-height));
```

Comment:
Sure but why make it depend on the window height? I'd rather have a min-height instead of seeing this in smaller resolutions:

![screen shot 2016-03-03 at 11 08 43](https://cloud.githubusercontent.com/assets/1078545/13491040/5cb36b2c-e130-11e5-83bf-96497c185fd6.png)